.storybook/themes/news.scss

```diff
@@ -456,6 +459,252 @@
 				),
 			),
 			blocks: (
+				article-body: (
+					font-family: var(--font-family-secondary),
+					font-size: var(--global-font-size-4),
+					components: (
+						link: (
+							color: var(--color-primary),
+						),
+						paragraph: (
+							display: block,
+							overflow: initial,
+						),
+						carousel: (),
+						carousel-actions: (
+							display: flex,
+						),
+						carousel-track: (
+							gap: initial,
+							max-width: initial,
+						),
+					),
+				),
+				article-body-link: (
+					color: var(--color-primary),
+				),
+				article-body-h2: (
+					font-size: var(--global-font-size-11),
+				),
+				article-body-h3: (
+					font-size: var(--global-font-size-10),
+				),
+				article-body-h4: (
+					font-size: var(--global-font-size-9),
+				),
+				article-body-h5: (
+					font-size: var(--global-font-size-7),
+				),
+				article-body-h6: (
+					font-size: var(--global-font-size-5),
+				),
+				article-body-blockquote: (
+					box-shadow: inset 5px 0 0 0 var(--global-neutral-4),
+					font-style: italic,
+					gap: var(--global-spacing-6),
+					padding: var(--global-spacing-4) 0 var(--global-spacing-4) var(--global-spacing-5),
+					text-align: left,
+					components: (
+						paragraph: (
+							font-size: var(--global-font-size-5),
+						),
+					),
+				),
+				article-body-children: (
+					margin-bottom: var(--global-spacing-5),
+				),
+				article-body-citation: (
+					font-family: var(--font-family-primary),
+					font-style: normal,
+					font-size: 16px,
```

Comment:
font-size available -- or is this in-progress? use a token here

Reply:
Updated